### PR TITLE
Levinska_CurrencyTests

### PR DESCRIPTION
**Pull request by sofiialevinska**
@adomanich iterations moved from Tests to MainPageBL

I was following the github instructions on how to resolve conflicts and unexpectedly merged the previous pull request into master...So this is new pull request to continue changes on CurrencyTests.

### CODE REVIEW

File: src/main/java/com/opencart/steps/MainPageBL.java [hunk 1]
```diff
@@ -6,13 +6,36 @@
...
 
+    /**
+     * Method finds all ProductContainers on the Page and checks
+     * if their Product Cost in specified currency is correct.
+     *
+     * @param currencies is a Map<String, String> with currencies Codes and Values that was read
+     *                   from the Admin Page.
+     */
+    public void verifyAllProductsPrices(Map<String, String> currencies) {
+        MainPageBL mainPageBL = new MainPageBL();
```
Comment (adomanich) [inline src/main/java/com/opencart/steps/MainPageBL.java]: You do not need to create a new instance, please, just call getHeaderBL()

File: src/test/java/com/opencart/tests/currency/ChangeCurrencyTest.java [hunk 2]
```diff
@@ -33,13 +33,6 @@ public void getCurrenciesValuesFromAdminPage() {
             "for currencies that are presented on admin page")
     public void ChangeCurrencyTest() {
```
Comment (adomanich) [inline src/test/java/com/opencart/tests/currency/ChangeCurrencyTest.java]: The last comment, if you have only one test in class, you do not need to move precondition logic to before class, so please, put it into the test

**Review by adomanich — APPROVED** [review]

**Comment by pekpytxd** [comment]
Nice one!

**Review by pekpytxd — APPROVED** [review]
Nice one!

**Review by cheesyy183 — APPROVED** [review]

**Pull request merged**